soql_suite_test.go

```diff
@@ -205,6 +205,10 @@ type QueryCriteriaWithDateTimeType struct {
 	CreatedDate time.Time `soql:"equalsOperator,fieldName=CreatedDate"`
 }
 
+type QueryCriteriaWithPtrDateTimeType struct {
+	CreatedDate *time.Time `soql:"equalsOperator,fieldName=CreatedDate"`
```

Comment:
May be just add another time field here and the existing testcase will cover both nil and not nil cases.

Reply:
Done.